ReleaseNotes.md

```diff
@@ -1076,7 +1076,7 @@ FEATURES:
 
 BUG FIXES:
 
- * Reinstalls an existing package if -version is passed (first surfaced in 0.9.8.7 w/NuGet 1.5) - [#9](https://github.com/chocolatey/chocolatey/issues/9)
+ * Reinstalls an existing package if --version is passed (first surfaced in 0.9.8.7 w/NuGet 1.5) - [#9](https://github.com/chocolatey/chocolatey/issues/9)
```

Comment:
Unfortunately this is not good. Way way back in time if you would have used 0.9.8.x and tried with `--parameter` it would have said "Computer says no"